index.js

```diff
@@ -1,9 +1,8 @@
+/* eslint-disable */
```

Comment:
- [ ] Please enable linters in your index.js file :blush: I know there are some errors. You can fix these errors by quick fix I have given below :arrow_down:
![quick-fix](https://user-images.githubusercontent.com/93039359/183714060-b1181352-0a43-495a-80d3-028357b46773.png)